scripts/ipfs-pi-stream/install

```diff
@@ -24,3 +24,14 @@ sudo systemctl enable process-stream
 
 # Add hourly job to clear out old data
 echo "41 * * * * $USER /usr/local/bin/ipfs repo gc" | sudo tee --append /etc/crontab
+
+# Install the ipfs video player
+mkdir "$BASE_DIR/tmp"
+git clone https://github.com/tomeshnet/ipfs-live-streaming.git "$BASE_DIR/tmp/live-streaming"
+cp -r "$BASE_DIR/tmp/live-streaming/terraform/shared/video-player" /var/www/html/video-player
+sed -i s#__IPFS_GATEWAY_SELF__#/ipfs/# /var/www/html/video-player/js/common.js
+sed -i s#__IPFS_GATEWAY_ORIGIN__#https://ipfs.io/ipfs/# /var/www/html/video-player/js/common.js
```

Comment:
I don't think origin would be ipfs.io here, it's localhost? And it shouldn't include the `/ipfs/`

Reply:
M3U8 file is still created with ipfs.io as the path.  This way it can be used "as is"

The script does a replace of `__IPFS_GATEWAY_ORIGIN__` to `__IPFS_GATEWAY_SELF__` so it needs to know what to replace.

Reply:
Ah I didn't realize the script creates with ipfs.io I thought it would just make localhost by default